### PR TITLE
feat(code-mappings): Add flag for auto-generated mappings

### PR DESCRIPTION
**Pull request by scttcper**
fixes https://getsentry.atlassian.net/browse/WOR-2269

### CODE REVIEW

**Review by wedamija — COMMENTED** [review]
The generate sql job is failing - check out the error it is emitting and you can follow the steps from that error to get your migration working

File: src/sentry/models/integrations/repository_project_path_config.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ class RepositoryProjectPathConfig(DefaultFieldsModel):
     source_root = models.TextField()
     default_branch = models.TextField(null=True)
     # Indicates if Sentry created this mapping
-    automatically_generated = models.BooleanField(default=False)
+    automatically_generated = models.BooleanField(null=True, default=False)
```
Comment (wedamija) [inline src/sentry/models/integrations/repository_project_path_config.py:20]: The column doesn't have to be null here. You can follow this guide to make the column not null with a default https://develop.sentry.dev/database-migrations/#adding-columns-with-a-default

**Review by wedamija — APPROVED** [review]

**Review by armenzg — APPROVED** [review]
Thank you @scttcper !

**Comment by armenzg** [comment]
Somebody has landed a migration with the same ID. If you follow [these steps](https://develop.sentry.dev/database-migrations/#merging-migrations-to-master) you will resolve it.

**Pull request merged**